client/signup/steps/site-options/index.scss

```diff
@@ -1,7 +1,8 @@
 @import '@wordpress/base-styles/_breakpoints.scss';
 @import '@wordpress/base-styles/_mixins.scss';
 
-.signup__step .is-site-options {
+.signup__step .is-site-options,
+.is-difm-options {
```

Comment:
Nit, why not add the same parent selector as above to maintain the same specificity :)